symphony-bdk-core/src/main/java/com/symphony/bdk/core/ServiceFactory.java

```diff
@@ -0,0 +1,64 @@
+package com.symphony.bdk.core;
+
+import com.symphony.bdk.core.api.invoker.ApiClient;
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.service.MessageService;
+import com.symphony.bdk.core.service.SessionService;
+import com.symphony.bdk.core.service.datafeed.DatafeedService;
+import com.symphony.bdk.core.service.datafeed.DatafeedVersion;
+import com.symphony.bdk.core.service.datafeed.impl.DatafeedServiceV1;
+import com.symphony.bdk.core.service.datafeed.impl.DatafeedServiceV2;
+import com.symphony.bdk.core.service.stream.StreamService;
+import com.symphony.bdk.core.service.user.UserService;
+import com.symphony.bdk.gen.api.AttachmentsApi;
+import com.symphony.bdk.gen.api.DatafeedApi;
+import com.symphony.bdk.gen.api.DefaultApi;
+import com.symphony.bdk.gen.api.MessageApi;
+import com.symphony.bdk.gen.api.MessageSuppressionApi;
+import com.symphony.bdk.gen.api.MessagesApi;
+import com.symphony.bdk.gen.api.PodApi;
+import com.symphony.bdk.gen.api.SessionApi;
+import com.symphony.bdk.gen.api.StreamsApi;
+import com.symphony.bdk.gen.api.UserApi;
+import com.symphony.bdk.gen.api.UsersApi;
+
+class ServiceFactory {
```

Comment:
Even if package private, let's mark this class as `@API(status = API.Status.INTERNAL)` + Javadoc comment